apps/network-alerts/js/index.js

```diff
@@ -14,14 +20,22 @@
   );
 
   /**
-   * Check if the id of the message in in CMAS specific range. Please ref
-   * http://www.etsi.org/deliver/etsi_ts/123000_123099/123041/11.06.00_60/
-   * ts_123041v110600p.pdf, chapter 9.4.1.2.2 Message Identifier for more
-   * details.
-   * @param {number} Message ID 
+   * Check if the id of the message in in CMAS specific range. Please ref link
+   * above for GSM[1] and CDMA[2]
+   *
+   * @param {number} Message
    */
-  function isEmergencyAlert(id) {
-    return (id >= 4370 && id < 4400);
+  function isEmergencyAlert(message) {
+    var messageId = message.messageId;
+    var cdmaServiceCategory = message.cdmaServiceCategory;
+
+    if (messageId) {
+      return (messageId >= 4370 && messageId < 4400);
+    } else if (cdmaServiceCategory) {
+      return (cdmaServiceCategory >= 4096 && cdmaServiceCategory <= 4351);
+    } else {
+      return false;
+    }
```

Comment:
Thanks for the suggestion! I think checking for the cdmaServiceCategory is null is not that necessary because null/undefined/0 for if statment are all equal to false. Unless the GSM message will have cdmaServiceCategory value other than null/undefined/0 or CDMA  message with cdmaServiceCategory = 0 , I think ``` if (cdmaServiceCategory)``` should be fairly enough here.